pom.xml

```diff
@@ -22,7 +22,7 @@ under the License.
 
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.apache.reef</groupId>
-    <version>0.10-incubating-SNAPSHOT</version>
```

Comment:
@dafrista Good catch. We have to change the version for all pom.xml files...

From the Apache release management page, "The most common system of versioning is based on major.minor.point where major, minor and point are all integers."

All the apache projects I checked use the above versioning. I think that it's better to fix the version number now. The next version we will use is 0.11.0-incubating-SNAPSHOT if we follow the guideline. 

@markusweimer Apache has a process that uses multiple steps (including mvn release): http://www.apache.org/dev/publishing-maven-artifacts.html. I'm following that to stage the release for a vote.